src/arch/ppc64.cpp

```diff
@@ -3,6 +3,11 @@
 #include <algorithm>
 #include <array>
 
+// For little endian 64 bit, sp + 32 + 8 regs save area + argX
+#define ARG0_STACK_LE 96
```

Comment:
Powerpc64, little endian, See Power Architecture 64-Bit ELF V2 ABI Specification
https://members.openpowerfoundation.org/document/dl/576
- See parameter save area
   - When the caller allocates the Parameter Save Area, it will always be automatically quadword aligned  because it must always start at SP + 32. **It shall be at least 8 doublewords in length.** If a function needs to pass more than 8 doublewords of arguments, the Parameter Save Area shall be large enough to spill all register-based parameters and to contain the arguments that the caller stores in it.

Reply:
Hm I'm a little confused. Seems like the ABI changed starting at power ISA 2.07B. So why is only LE using the new ABI?

Won't block this PR on these questions. Realizing this was an issue is a big win already.